eng/common/templates/jobs/publish.yml

```diff
@@ -26,6 +30,7 @@ jobs:
       --os-type '*'
       --architecture '*'
       --repo-prefix $(publishRepoPrefix)
+      --image-info $(artifactsPath)/image-info.json
```

Comment:
Thoughts on making this optional and only used in this repo?

Reply:
Unless there's a need for it, I'd prefer to keep it consistent.  I think we should strive for no special conditions unless they're really necessary so that we can more easily determine whether a change breaks something.  I want to avoid situations where the pipeline works in one GH repo but not in another one.

Reply:
I don't have a strong argument against that.  If this is the direction, then is there any value in having the CopyAcrImagesCommand support having no image-info.json?  It seems like dead code that is adding complexity albeit minor, to the codebase.

Reply:
Keeping it in there simplifies running the command locally.  So I think there's value in that.

Reply:
And it might be a good fallback to have just in case things aren't working properly when making use of the image info.  If we're trying to get a release out the door and the use of image info is causing errors for whatever reason, we can quickly change the pipeline to not pass in the image info.